### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by myshevts**
### Details:
 - CPU latency case description update
 - Lateny tips for the Opt Guide and general brushing

### CODE REVIEW

**Review by andrew-zaytsev — DISMISSED** [review]

File: docs/optimization_guide/dldt_optimization_guide.md [hunk 1]
```diff
@@ -55,14 +55,16 @@ In contrast, for the latency-oriented tasks, the time to a single frame is more
...
+> **NOTE**: The [Benchmark App](../../inference-engine/samples/benchmark_app/README.md) sample also support batching (automatically packing multiple input images into a single request). However, high batch size results in a latency penalty. So for more real-time oriented usages, lower batch sizes (as low as a single input) are usually used. Still, devices like CPU, Intel&reg; Movidius&trade; Myriad&trade; 2 VPU, Intel&reg; Movidius&trade; Myriad&trade; X VPU, or Intel® Vision Accelerator Design with Intel® Movidius™ VPU require a number of parallel requests instead of batching to leverage the performance. Running the multiple requests should be coupled with a device configured to the corresponding number of "streams", for example the CPU streams are explained in the <a href="#cpu-streams">corresponding</a> section.
+
+[OpenVINO Deep Learning Workbench tool](https://docs.openvinotoolkit.org/latest/workbench_docs_Workbench_DG_Introduction.html) allows to build throughput versus latency charts for different number of streams/requests/batch to find a performance sweet-spot.
```
Comment (aalborov) [inline docs/optimization_guide/dldt_optimization_guide.md:60]:
```suggestion
[OpenVINO™ Deep Learning Workbench tool](https://docs.openvinotoolkit.org/latest/workbench_docs_Workbench_DG_Introduction.html) provides throughput versus latency charts for different numbers of streams, requests, and batch sizes to find the performance sweet spot.
```

File: docs/optimization_guide/dldt_optimization_guide.md [hunk 2]
```diff
@@ -77,7 +79,7 @@ You need to build your performance conclusions on reproducible data. Do the perf
 -	If the warm-up run does not help or execution time still varies, you can try running a large number of iterations and then average or find a mean of the results.
 -	 For time values that range too much, use geomean.
 
-Refer to the [Inference Engine Samples](../IE_DG/Samples_Overview.md) for code examples for the performance measurements. Almost every sample, except interactive demos, has a `-ni` option to specify the number of iterations.
+Refer to the [Benchmark App](../../inference-engine/samples/benchmark_app/README.md) for code examples for the performance measurements. Almost every sample, except interactive demos, has a `-ni` option to specify the number of iterations.
```
Comment (aalborov) [inline docs/optimization_guide/dldt_optimization_guide.md:82]:
```suggestion
Refer to the [Benchmark App](../../inference-engine/samples/benchmark_app/README.md) for code examples of performance measurements. Almost every sample, except interactive demos, has the `-ni` option to specify the number of iterations.
```

**Review by aalborov — DISMISSED** [review]

**Comment by myshevts** [comment]
I have resolved all comments. Can you pls merge?

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@myshevts could you please manually port this to master?

**Pull request merged**